src/Microsoft.Identity.Web.TokenCache/MsalAbstractTokenCacheProvider.Logger.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web.TokenCacheProviders
+{
+    /// <summary>
+    /// LoggingMessage class for MsalAbstractTokenCacheProvider.
+    /// </summary>
+    public abstract partial class MsalAbstractTokenCacheProvider
+    {
+        /// <summary>
+        /// LoggingMessage class for MsalAbstractTokenCacheProvider.
+        /// </summary>
+        private static class Logger
+        {
+            private static readonly Action<ILogger, string, string, Exception> s_cacheDeserializationError =
+                LoggerMessage.Define<string, string>(
+                    LogLevel.Warning,
```

Comment:
Although this is an exception, setting log level to Warning, since the execution is continued.